guides/common/modules/con_how-to-configure-inter-server-sync.adoc

```diff
@@ -29,7 +29,7 @@ This server consumes content from the Red Hat Content Delivery Network (CDN) or
 * The downstream {ProjectServer} can communicate with a connected upstream {ProjectServer} over an internal network.
 
 ifndef::satellite[]
-image::common/foreman_iss_disconnected_non_airgapped.png[title="The {Project} ISS Disconnected Scenario", alt="The {Project} ISS Disconnected Scenario"]
+image::images/common/foreman_iss_disconnected_non_airgapped.png[title="The {Project} ISS Disconnected Scenario", alt="The {Project} ISS Disconnected Scenario"]
```

Comment:
@maximiliankolb FYI - currently, you have Foreman image in the orcharhino build here.

Reply:
This is OK for me because we take care of images in our downstream branch. Sadly asciidoctor cannot handle `.svg` images in the current way we build the docs using GHA & I didn't bother fixing this yet.